### PR TITLE
10# Game class

### PR DESCRIPTION
**Pull request by LuckyCode7**

### CODE REVIEW

**Comment by LuckyCode7** [comment]
too many "mistakes removed" :), but there was a problem with loading the text line from file

File: src/Game.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        if(!(inputValidation->checkInputData()) && oneLine != "")
+        {
+            throw IncorrectInputData(oneLine);
+        }
+        else setGameStatistic(inputValidation, oneLine);
+    }
+    correctnessOfInputData = true;
+    }
+    else throw InvalidFile(pathToFile);
+}
```
Comment (wektor1) [inline src/Game.cpp]: A bit long function, 'while' loop  can be moved to another function

File: results/Game4.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Lucky:x|x|x|x|x|x|x|x|x|x||x
-Kate:x|x|x|x|x|x|x|x|x|x||x
\ No newline at end of file
+Lucky:x|x|x|x|x|x|x|x|x|--||
+Kate:x|x|x|x|x|x|x|x|x|-/||1
```
Comment (wektor1) [inline results/Game4.txt]: Changes of "not code" files should be in separate pull request
